src/main/java/de/qabel/desktop/ui/about/AboutController.java

```diff
@@ -101,6 +117,31 @@ private void indexJarNames() throws ParserConfigurationException, IOException, S
         }
     }
 
+    private void activateButtons() {
+        if (thanksFileContent == null) {
```

Comment:
This case is impossible.
Even if the contents have not been loaded, an `@Inject` will never work with `null`. It would lead to an InvalidStateException when initializing the Controller. One reason to not use a null but either an empty string or a ValueObject for this on error.